test/codeu/chat/util/TokenizerTest.java

```diff
@@ -0,0 +1,26 @@
+package codeu.chat.util;
+
+import java.io.IOException;
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public final class TokenizerTest {
+    @Test
+    public void testWithQuotes() throws IOException {
+        final Tokenizer tokenizer = new Tokenizer("hello world how are you");
+        assertEquals(tokenizer.next(), "hello");
+        assertEquals(tokenizer.next(), "world");
+        assertEquals(tokenizer.next(), "how");
+        assertEquals(tokenizer.next(), "are");
+        assertEquals(tokenizer.next(), "you");
+        assertEquals(tokenizer.next(), null);
+    }
+
+    @Test
+    public void testWithNoQuotes() throws IOException {
```

Comment:
The code is well designed here. Tests are broken up into separate functions with descriptive names. You could include a few more unit tests in this file just to test more complex edge cases.